doc/nrf/ug_thread_supported_features.rst

```diff
@@ -30,6 +30,14 @@ The OpenThread implementation of the Thread protocol supports all features defin
 
   * Service Registration Protocol (SRP) client
 
+* Features introduced with Thread 1.3.0:
+
+  * Service Registration Protocol (SRP) client
+
+In |NCS|, you can choose which version of the Thread protocol to use in your application.
+By default, |NCS| supports Thread 1.3, which includes support for Thread 1.2.
+You can enable and configure any Thread version by using :ref:`dedicated options <thread_ug_thread_specification_options>`.
```

Comment:
The section linked only mentions versions 1.1 and 1.2, and by the looks of it, it should also mention 1.3. However, maybe we can remove the list of supported features from there and just point to the list in this file instead.

Reply:
I missed to pick some changes in `ug_thread_configuring.rst`. Now it should be better.